src/lib/pages/backoffice/Loan/LoanSearch/LoanSearch.js

```diff
@@ -15,7 +15,7 @@ import { NewButton } from '@components/backoffice/buttons/NewButton';
 import { loanApi } from '@api/loans';
 import { BackOfficeRoutes } from '@routes/urls';
 import history from '@history';
-import LoanList from './LoanList';
+import { LoanList } from '@components/backoffice/LoanList';
```

Comment:
the same comment for the LoanList component:
please move it to `modules/Loan/backoffice/` 
the principle is, when it is a simple UI component then it goes to the `@components`
when it is connected with a record type, expects specifically data from the backend, then it should go to `@module`